docs/using.md

```diff
@@ -186,18 +186,19 @@ Use the following example to understand the syntax and strucutre of a Git Resour
     apiVersion: pipeline.knative.dev/v1alpha1
     kind: TaskRun
     metadata:
-    name: build-push-task-run
-    namespace: default
+      name: build-push-task-run
+      namespace: default
     spec:
-        taskRef:
-            name: build-push-task
-        inputs:
-            resourcesVersion:
-            - resourceRef:
-                name: wizzbang-git
-                version: HEAD
-        outputs:
-            artifacts:
-            - name: builtImage
-                type: image
+      taskRef:
+        name: build-push-task
+      inputs:
+        resourcesVersion:
+        - resourceRef:
+            name: wizzbang-git
+            apiVersion: HEAD
+      outputs:
+        resources:
+        - name: builtImage
+          type: image
+      # serviceAccount: build-bot
```

Comment:
if you want to add serviceAccount here can you add not make commented out and put a comment that it is optional

Reply:
OK, i will fix it.

Reply:
I have fixed it, thank you.